README.md

```diff
@@ -139,17 +139,16 @@ Alert Engine Advanced Configuration Snippet (Safety Valve) for alert-engine.yml`
 
 ## Running the application manually on a cluster
 
-Remove the `.template` suffix from these two files under the local directory.
+Remove the `.template` suffix from these two files under the `bin` directory.
 - `cp env.sh.template env.sh`
-- `cp jaas.conf.template jaas.conf`
```

Comment:
maybe keep the reference to `cp jaas.conf.template jaas.conf` since it mentions two files

Reply:
This step shouldn't be needed anymore, the jaas.conf is generated by the script from the other environment variables.